bio2zarr/vcf2zarr/icf.py

```diff
@@ -458,6 +486,27 @@ def sanitise_value_int_2d(buff, j, value):
         buff[j, :, : value.shape[1]] = value
 
 
+def compute_laa_field(variant) -> list[list[int]]:
+    sample_count = variant.num_called + variant.num_unknown
+    laa_val = [set() for _ in range(sample_count)]
+
+    if "GT" in variant.FORMAT:
+        for sample_index, genotype in enumerate(variant.genotypes):
```

Comment:
This will be too slow for production use I'm afraid - we can't have any Python loops over the samples, which is O(10^6). Can we do the same thing using numpy operations?

For example, would something like ``np.bincount(variant.genotypes[some_selector])`` do the trick?

Also, might as well count the reference alleles as well, as we're at it?